wiquery-jquery-ui/src/main/resources/org/wicketstuff/wiquery/ui/autocomplete/wiquery-autocomplete.js

```diff
@@ -43,7 +43,7 @@
 	
 			} else {
 			  	var find = null;
-			  	var source = $(event.target).autocomplete('option', 'source');
+			  	var source = new String($(event.target).autocomplete('option', 'source'));
```

Comment:
I am not sure this is the proper fix.
Looking at the $.each below the `source` is expected to be an Array of Objects (https://api.jqueryui.com/autocomplete/#option-source)